main/EDInitMod.F90

```diff
@@ -487,7 +487,7 @@ subroutine init_patches( nsites, sites, bc_in)
           start_patch = 1   ! start at the first vegetated patch
           if(hlm_use_nocomp.eq.itrue)then
              num_new_patches = numpft
-             if(hlm_use_sp.eq.itrue)then
+             if( (hlm_use_sp.eq.itrue) .or. (hlm_use_fixed_biogeog .eq.itrue) )then
```

Comment:
We should be able to simplify this to just check fixed biogeog here since sp mode requires fixed biogeog:
https://github.com/NGEET/fates/blob/e1cf819ec3158b3a02cd693c73680dbca3cb4924/main/FatesInterfaceMod.F90#L1560-L1563

Reply:
Right.  I suspected that but wasn't completely sure that the code enforced it, so thanks for checking.  I'll update the code accordingly.